app/code/Magento/GroupedProduct/view/adminhtml/templates/catalog/product/composite/fieldset/grouped.phtml

```diff
@@ -6,40 +6,44 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
 
-<?php /* @var $block \Magento\GroupedProduct\Block\Adminhtml\Product\Composite\Fieldset\Grouped */ ?>
+<?php
+/**
+ * @var $block \Magento\GroupedProduct\Block\Adminhtml\Product\Composite\Fieldset\Grouped
```

Comment:
it seems like you missed adding escaper here

Reply:
fixed